primer3/argdefaults.py

```diff
@@ -42,5 +53,168 @@ class Primer3PyArguments:
     temp_only: int = 0
     calc_type_wrapper = 'ANY'
 
+    dmso_conc: float = 0.0
+    dmso_fact: float = 0.6
+    formamide_conc: float = 0.8
+    annealing_temp_c: float = 50.0  # 70.0 or 120 in primer3 source
+
     def todict(self) -> dict:
         return dataclasses.asdict(self)
+
+
+# ~~~~~~~ RUDIMENTARY PRIMER3 MAIN WRAPPER (see Primer3 docs for args) ~~~~~~ #
+def format_boulder_io(p3_args: Dict[str, Any]) -> bytes:
+    '''Convert argument dictionary to boulder formatted bytes
+
+    Args:
+        p3_args: primer3 arguments to format boulder style
+
+    Returns:
+        Boulder formatted byte string
+    '''
+    class TransformT:
```

Comment:
I'm sure we'll document this elsewhere, but it might be good to have inline comments w/ examples of each string format?

Reply:
I am am regressing this and moving `wrap()` and `unwrap()` code from `_simulatedbindings.py` to `argdefaults` to use in that pattern instead, since it pre-existed

Reply:
> Would it be simpler to just parse the `BoulderIO` output string _in lieu_ of `pdh_design_output_to_dict`? Seems like the inverse of the pattern in `format_boulder_io` could work well

There's different keys in the output, but we could definitely do something like that.